action_server/src/action_server/client.py

```diff
@@ -139,6 +139,10 @@ def send_task(self, semantics):
             self.cancel_all()
             raise KeyboardInterrupt
 
+        if not isinstance(result, action_server_msgs.msg.TaskResult):
+            rospy.logerr("Result not instance of 'action_server_msgs.msg.TaskResult', but {}".format(type(result)))
+            return False
```

Comment:
Docstring mentions the method returns a bool and a str message.

Reply:
Fixed this.